.changeset/five-ravens-own.md

```diff
@@ -0,0 +1,15 @@
+---
```

Comment:
This doesnt matter much here, but I believe those should get their own changesets. Think about those as separate entries - I think making them separate will also generate slightly different changelog. This now is a single multi-line entry, whereas they could be punctuated (if you would use separate changesets). Using separate changesets has also other benefits - including more flexibility because separate changesets can have different "types" (major/minor/patch) and when changelog is generated those "types" are grouped together.

"Prerelease" changesets are kept when releasing prerelease versions (because when exiting the prerelease stage all of them are used to generate a single "final" changelog entry), so keeping them separate will make reviewing them easier when exiting the prerelease stage (we might want to remove no longer relevant stuff there, so they dont get listed in the final changelog entry).